lib/main_common.pm

```diff
@@ -538,6 +538,8 @@ sub load_extra_tests {
         # test, checking the wifi applet, would make sense in other DEs as
         # well
         if (check_var('DESKTOP', 'gnome')) {
+            loadtest 'x11/yast2_lan_restart';
+            loadtest 'x11/yast2_lan_devices';
             loadtest 'x11/yast2_lan_restart';
```

Comment:
exactly, is a question that I was about to arise, in the original code the only difference that I found is that the first one the hw device is changed to "dyn0" and the second to "sta0". I was thinking to use a random string to generate the name of the device "xyz0" to respect the original functionality, but it might not be needed as you pointed. The second yast2_lan_restart looks independent because in yast2_lan_devices the devices are added and removed. I can remove, it seems to not provide any extra testing, right?